docker/dockerfiles/Dockerfile

```diff
@@ -102,9 +102,9 @@ RUN chown -R mysql:mysql /var/lib/mysql \
 RUN apt-get update && apt-get -y install parallel libmagickwand-dev imagemagick
 RUN mv /usr/bin/parallel /usr/bin/gnu_parallel
 
-RUN curl -sSL -o /tmp/pdftk-java_3.0.2-2_all.deb https://mirrors.kernel.org/ubuntu/pool/universe/p/pdftk-java/pdftk-java_3.0.2-2_all.deb \
-  && DEBIAN_FRONTEND=noninteractive dpkg -i /tmp/pdftk-java_3.0.2-2_all.deb || apt-get -fy install \
-  && rm -rf /tmp/pdftk-java_3.0.2-2_all.deb
+RUN curl -sSL -o /tmp/pdftk-java_3.0.9-1_all.deb https://mirrors.kernel.org/ubuntu/pool/universe/p/pdftk-java/pdftk-java_3.0.9-1_all.deb \
```

Comment:
do you know why this needs to be updated alongside node?

Reply:
The previous download link was 404-ing, so I updated it to the latest version on their site. Not sure how often a mirror like that will fail...

Reply:
so, not a strict dependency on node, more because that docker image hadn't been updated in a year or so probably